charts/pelorus/templates/prometheus-scrape-config-secret.yaml

```diff
@@ -1,4 +1,29 @@
 {{- define "prometheus_scrape_config"}}
+  {{- with .Values.federate_openshift_monitoring }}
+  {{- if .enabled }}
+- job_name: 'federate'
+  scrape_interval: 15s
+  honor_labels: true
+  metrics_path: '/federate'
+  params:
+    'match[]':
+    {{- if .metrics_filter }}
+      {{ .metrics_filter | toYaml }}
+    {{- else }}
+      - '{job="kube-state-metrics"}'
+      - '{job="openshift-state-metrics"}'
+    {{- end }}
+  static_configs:
+    - targets:
+      - 'prometheus-k8s.openshift-monitoring.svc:9091'
```

Comment:
Could we make a comment it's static?

https://github.com/openshift/cluster-monitoring-operator/blob/master/assets/prometheus-k8s/prometheus.yaml#L160

Reply:
Is that what you meant?